crates/sparrow-runtime/src/metadata/raw_metadata.rs

```diff
@@ -171,6 +181,7 @@ impl RawMetadata {
     /// Create a `RawMetadata` from a Pulsar topic.
     pub(crate) async fn try_from_pulsar(
         config: &PulsarConfig,
+        should_include_publish_time: bool,
```

Comment:
I think this can just be `include_publish_time` since it controls the behavior. I don't think `should_` adds much.